### PR TITLE
missing last section of code

### PR DESCRIPTION
**Pull request by lanceaune**
I will make sure to complete all assignments going forward. Apologies for my last 2 not being fully completed.

### CODE REVIEW

**Comment by mbMosman** [comment]
If you haven't finished this up yet, keep going on it to get it finished.


Mary Mosman

Computer Careers Faculty

Brooklyn Park Campus, 9000 Brooklyn Boulevard, Brooklyn Park, MN 55445

Eden Prairie Campus, 13100 College View Drive, Eden Prairie, MN 55347

HennepinTech.edu<https://mail.hennepintech.edu/owa/redir.aspx?SURL=X9_UXdraVORHgBEDAbJjlW7lFDPLHRjN-BS9r4ndo2C1J6JJO7nSCGgAdAB0AHAAcwA6AC8ALwBoAGUAbgBuAGUAcABpAG4AdABlAGMAaAAuAGUAZAB1AC8A&URL=https%3a%2f%2fhennepintech.edu%2f>

Support the HTC Foundation by using AmazonSmile<https://mail.hennepintech.edu/owa/redir.aspx?SURL=mTQW3FiySLMvYJeLHK4cUN5xDY1QSOqoOR3YmKzO9qu1J6JJO7nSCGgAdAB0AHAAOgAvAC8AcwBtAGkAbABlAC4AYQBtAGEAegBvAG4ALgBjAG8AbQAvAGMAaAAvADQAMQAtADEAOQA0ADMAMQA0ADkA&URL=http%3a%2f%2fsmile.amazon.com%2fch%2f41-1943149>

________________________________

**Comment by lanceaune** [comment]
Hi Mary,

Can you direct me towards where I could find some example code or other code to help me figure out how to do the last section.

Thanks
________________________________

**Comment by mbMosman** [comment]
It would be easier to do this if you'd say exactly what you are looking for...  I'm assuming you mean so that you don't have many images showing down the side of the display?  You need to remove the previous image before appending in a new one.  There were two ways to do this included in the lecture slides.

**Comment by lanceaune** [comment]
Hi Mary,

I think I may have figured out how to clear the previous image when another menu item is clicked. I will test it out when I am home. I will keep you posted.

- Lance
________________________________

**Comment by lanceaune** [comment]
Hi Mary,


I figured it out, the missing code has been added now. I pushed the completed project to the directory.

________________________________

File: index.html [hunk 1]
```diff
@@ -1,52 +1,58 @@
...
+    <header>
+        <h1>Moonflower's Coffee Shop</h1>
+    </header>
+
+    <main class="container-fluid">
+
+        <row class="col-xs-12 col-md-8">
+            <section id="menu">
+                <h2>Menu</h2>
+                <h4>Click on an item to see the image!</h4>
```
Comment (mbMosman) [inline index.html]: This should be paragraph text and should be added through JavaScript. The reason for that being it won't work unless JavaScript is running and enabled in the browser.

**Comment by mbMosman** [comment]
Nice work with the images.

**Pull request closed without merging**